### PR TITLE
Reformat LMP Field

### LINKED ISSUE #280: New pregnancy form: Adjust LMP to be weeks/months dropdowns instead of open text field
Raised in #273 -- Our current form in `views/pregnancies/_new_pregnancy` is as follows:

screen shot 2016-05-19 at 10 38 31 am

Let's change the LMP representation here to look more like what's in `views/pregnancies/_patient_dashboard` in the edit view, where weeks and days are separate dropdowns. This will ensure we stay consistent and avoid confusion on data entry.

### PR DESCRIPTION
**Pull request by CamLatimer**
I rule and have completed some work on Case Manager that's ready for review!

This pull request makes the following changes:
Changed LMP fields to look more like what's in views/pregnancies/_patient_dashboard in the edit view

It relates to the following issue #s: 
* Fixes #280

### CODE REVIEW

**Comment by colinxfleming** [comment]
@CamLatimer I'm going to use this pull request as a way to train @ajohnson052 how to review PRs. She and I will tag team it shortly. Thanks much!

**Comment by CamLatimer** [comment]
Got it. No prob!

File: README.md [hunk 1]
```diff
@@ -67,22 +67,27 @@ If you don't currently have Rails installed (or are on Windows), Cloud9 makes th
 
 ### Docker
 
+<<<<<<< HEAD
```
Comment (ajohnson052) [inline README.md]: Can you take a look at this merge conflict? Colin made a couple small changes (just that spelling error and the added link) so go ahead and keep those. Thanks!

File: app/views/pregnancies/_new_pregnancy.html.erb [hunk 2]
```diff
@@ -11,10 +11,10 @@
   <% end %>
 
   <div class="col-md-2">
-    <%= f.text_field :last_menstrual_period_weeks, label: "LMP" %>
+    <%= f.select :last_menstrual_period_weeks, options_for_select(weeks_options, pregnancy.last_menstrual_period_weeks ), label: 'LMP at intake', autocomplete: 'off' %>
   </div>
   <div class="col-md-2">
-    <%= f.date_field :initial_call_date, label: "Initial Call Date" %>
+    <%= f.select :last_menstrual_period_days, options_for_select(days_options, pregnancy.last_menstrual_period_days ), label: '.', autocomplete: 'off' %>
```
Comment (ajohnson052) [inline app/views/pregnancies/_new_pregnancy.html.erb]: This looks great! The weeks and days fields should work well, but can you make sure that you keep in the field for :initial_call_date?

**Comment by ajohnson052** [comment]
Thanks for your work on this issue Cam! There's just one small merge conflict to fix, and we'll want to make sure that we aren't removing the initial call field. Then we'll be all set! On a side note, I'm going to make a new issue for coming up with a new label for LMP days both here and in the patient dashboard.

**Comment by colinxfleming** [comment]
New issue to remove the period in the LMP label represented in #321 . Thanks @ajohnson052 !

**Comment by colinxfleming** [comment]
The majority of conflicts are resolved here; @rekhers is going to pick up this thread and put a nail in it. I'm going to merge after the CI finishes. 

Thanks @CamLatimer and @ajohnson052 !

**Pull request merged**